BaseTools/Source/C/GenFw/Elf64Convert.c

```diff
@@ -2398,6 +2400,10 @@ WriteExport64 (
   UINT16                              Index;
   UINT8                               *Tdata = NULL;
 
+  if (mExportSymNum == 0) {
+    Error (NULL, 0, 3000, "Invalid", "%s has no export symbols.", mInImageName);

```

Comment:
Could this error be made explicit? Mentioning that prm parameter was specified but there are not exports available for it?

Reply:
I agree.  I will review all the new error messages in this PR to make sure they are useful.